### PR TITLE
Add stylesheets features

### PR DESCRIPTION
**Pull request by aubinlorieux**
As comment on #19 i've implemented the possibility to switch original stylesheets:
- Using [commander](https://github.com/tj/commander.js) module for cli.
- Add bin/ folder for cli and create a module for displaying markdown. I see possible evolution in the future with more command (ex: pdf export)
- I made some refactoring globally
- i tried to keep the same coding style (no semi-colon, use of const)

Waiting for reviews and comments

### CODE REVIEW

**Comment by aubinlorieux** [comment]
I think a `git rebase` would be great to clean up commits

**Comment by maxkueng** [comment]
Awesome!

I've recently been using the amazing [rc](https://www.npmjs.com/package/rc) module which could be an alternative to commander in this case.
`rc` is a configuration loader that checks for config files in all common locations (such as `~/.vmdrc` for example). But it allows you to override configuration parameters using command-line options. So the `style` config could be overridden with `--style=molokai`.
I think most users have one preferred style that doesn't change each time they run `vmd`. So, allowing to make those settings permanent (i.e. a config file) would probably be better.

Also, highlight.js provides [a number of styles](https://github.com/isagalaev/highlight.js/tree/master/src/styles). Instead of always providing the full path to a CSS file in the `--style` option, maybe it would be cool if one could just provide the name of the style, and if it doesn't exist or the option value is a valid path to a file it would include an actual external CSS file.

**Comment by aubinlorieux** [comment]
Yes it's definitely possible and a great idea and do not seems much of a work to implement. i will look at all this when i've time.

File: server.js [hunk 1]
```diff
@@ -1,48 +1,58 @@
...
 
 // Quit when all windows are closed.
 app.on('window-all-closed', function () {
   if (process.platform !== 'darwin') app.quit()
-})
+});
 
 app.on('ready', function () {
-  window = new BrowserWindow({
+  var win = new BrowserWindow({
```
Comment (yoshuawuyts) [inline server.js]: this won't work since properties must be bound to `window`

**Comment by yoshuawuyts** [comment]
Nice work, really appreciate the effort you've put into this! However, this patch introduces some aspects I don't agree with. 
- In order to prevent discussion about coding style I've adopted [`standard`](http://ghub.io/standard) for `vmd`.
- I'd rather use `minimist` than `commander` since it's much simpler.
- I'd preferably not pull in `underscore` / `lodash` into `vmd`, if you need a specific function it's better to pull in a more dedicated module instead.

If you want I can cherry pick your changes and merge them in a separate PR, but if you want to add these in yourself rebasing onto master would be a good place to start. Thanks again! :tada:

**Comment by maxkueng** [comment]
@yoshuawuyts I's suggest uding `rc`. `rc` uses `minimist` underneeth. It's like `minimist` but it allows to make options permenant in the form of a config file.

**Comment by yoshuawuyts** [comment]
@maxkueng oh yeah, I agree that that's better for this use case. Nice one.

File: server.js (hunk 1 above)
Comment (aubinlorieux) [inline server.js]: yes totally i should have declare  `win`  in the global scope before assigning `BrowserWindow`. Anyway i prefere to not use the `window` keyword.

**Comment by aubinlorieux** [comment]
Ok! I will look at all this and install `standard`. Are you all ok with `rc` ?

**Comment by maxkueng** [comment]
@aubinlorieux You can just `npm test` and it will run the code against `standards` for you. No  need to install separately.

I vote for `rc`. The only disadvantage is that it [doesn't support aliases out-of-the-box](https://github.com/dominictarr/rc/issues/39) because it would break compativility with other option parsers. But you could provide your own instance of `minimist` (see ["Pass in your own parser" section](https://github.com/dominictarr/rc#pass-in-your-own-parser)) with aliases.

**Comment by yoshuawuyts** [comment]
There's been little movement in this lately. Closing for now; feel free to reopen if there's any developments.

**Pull request closed without merging**